extension/popup/popupView.jsx

```diff
@@ -558,6 +559,15 @@ this.popupView = (function() {
               </span>
             </div>
           ) : null}
+          {previous ? (
+            <div id="search-show-previous">
+              Say <strong>previous result</strong> to view: <br />
+              <strong id="search-show-previous-title">{previous.title}</strong>
+              <span id="search-show-previous-domain">
+                {new URL(previous.url).hostname}
+              </span>
+            </div>
+          ) : null}
```

Comment:
We probably don't have to show anything about the previous result – the user already saw it, and should have some familiarity with it. We might want to suggest that "previous" is an option to say, but for now I think we should just leave it out of the view entirely.

Reply:
Ok. :+1: